setup.cfg

```diff
@@ -0,0 +1,5 @@
+[pylama]
+ignore=W391
```

Comment:
If this file can contain comments, would be good to note here that W391 is blank line at end of file